canonical-web-and-design/maas-ui
# refactor(ui): update notification to use a slice

## Done

- Replace the notification actions and reducers with a slice.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Check that you can see and dismiss notifications (I used `make harness` to create some notifications).

## Fixes

Fixes: canonical-web-and-design/maas-ui#1658
